pkg/tfbridge/provider.go

```diff
@@ -416,6 +417,25 @@ func (p *Provider) Configure(ctx context.Context,
 	return &pulumirpc.ConfigureResponse{}, nil
 }
 
+var requiredFieldRegex = regexp.MustCompile("\"(.*?)\": required field is not set")
```

Comment:
Perhaps a reference to the string defined here which this regexp should be kept in sync with: https://github.com/hashicorp/terraform/blob/7f5ffbfe9027c34c4ce1062a42b6e8d80b5504e0/helper/schema/schema.go#L1356.